### PR TITLE
Allow any characters in ReportAs

### PR DESCRIPTION
**Pull request by abaranch**

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
:shipit:

**Comment by tokaplan** [comment]
`shipit`

File: Src/PerformanceCollector/Unit.Tests/PerformanceCollectorModuleTests.cs [hunk 1]
```diff
@@ -281,25 +283,25 @@ public void CustomCountersSanitizingTest()
...
+                    Assert.AreEqual(@" Counter5", collector.Counters[6].Item1.ReportAs);
 
                     // unicode-only reportAs values are converted to "random" strings
                     Assert.AreEqual(@"\Категория6\Счетчик6", collector.Counters[7].Item1.OriginalString);
-                    Assert.AreEqual(@"Performance counter A", collector.Counters[7].Item1.ReportAs);
+                    Assert.AreEqual(@"Только юникод первый", collector.Counters[7].Item1.ReportAs);
 
                     Assert.AreEqual(@"\Категория7\Счетчик7", collector.Counters[8].Item1.OriginalString);
-                    Assert.AreEqual(@"Performance counter B", collector.Counters[8].Item1.ReportAs);
+                    Assert.AreEqual(@"Только юникод второй", collector.Counters[8].Item1.ReportAs);
```
Comment (dnduffy) [inline Src/PerformanceCollector/Unit.Tests/PerformanceCollectorModuleTests.cs]: Just curious why these are 1st and 2nd unicode strings when the test sequence is at 6 & 7? ;) (no need to change it)

**Comment by dnduffy** [comment]
:+1:

**Pull request merged**